homework-todo.ts

```diff
@@ -0,0 +1,97 @@
+interface INote {
```

Comment:
There should be two types of notes - with the "requiresConfirmation" field and without it. At least it should be an optional field. Please, do not attempt to implement the simpler solution.